cfme/tests/infrastructure/test_vm_reconfigure.py

```diff
@@ -43,7 +43,7 @@ def reconfigure_vm(vm, config):
                 .format(vars(config.hw), config.disks))
 
 
-@pytest.fixture(scope='module')
+@pytest.fixture(scope='function')
```

Comment:
Required: remove (scope='function') as  @pytest.fixture as by default function scoped

Reply:
@jan-zmeskal It would be nice if you add a description in this fixture that explains why it should a function scoped.

Reply:
@quarckster Done